### PR TITLE
Add DLLocalNotifications

### PR DESCRIPTION
**Pull request by d7laungani**
Was wondering if it was ok to add DLLocalNotifications to the list. Since it does not fit under push [edited] notifications, I created a new high-level category called notifications and moved push notifications underneath that and added a local notifications sub-category.
Thanks!

## Project URL
https://github.com/d7laungani/DLLocalNotifications

## Description
Easily create Local Notifications in swift - Wrapper of UserNotifications Framework
 
## Checklist

- [x] Only one project/change is in this pull request
- [X] Addition in chronological order (bottom of category)
- [X] Supports iOS 9 or later
- [X] Supports Swift 3
- [X] Has a commit from less than 2 years ago
- [X] Has a **clear** README in English

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Found 3 link issues</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Too many changes (when adding, please keep it to one project per Pull Request)</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
2418 | [503](https://httpstatuses.com/503) | https://www.amazon.com/Programming-Objective-C-6th-Developers-Library/dp/0321967607 
2421 | [503](https://httpstatuses.com/503) | https://www.amazon.com/Core-Data-iOS-Data-Driven-Applications/dp/0321670426 
2422 | [503](https://httpstatuses.com/503) | https://www.amazon.com/Cocoa-Design-Patterns-Erik-Buck/dp/0321535022 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by d7laungani** [comment]
I can re-submit in a different section if you find that it fits in a different category.

**Comment by dkhamsing** [comment]
@d7laungani we'll review this, thanks

**Comment by lfarah** [comment]
I liked the 2 sections, but I'd put push notifications first since there are more libs on that category.

File: README.md [hunk 1]
```diff
@@ -918,6 +919,39 @@ Also see [push notifications](#push-notifications)
 * [SwiftyZeroMQ](https://github.com/azawawi/SwiftyZeroMQ) - ZeroMQ Swift Bindings for iOS, macOS, tvOS and watchOS. :large_orange_diamond: ⌚
 * [Nikka](https://github.com/JustaLab/Nikka) - A super simple Networking wrapper that supports many JSON libraries, Futures and Rx :large_orange_diamond: ⌚
 
+## Notifications
+
+#### Local Notifications
+* [DLLocalNotifications](https://github.com/d7laungani/DLLocalNotifications) -  Easily create Local Notifications in swift - Wrapper of UserNotifications Framework. :large_orange_diamond:
+
+
```
Comment (lfarah) [inline README.md]: Please remove this extra space

File: README.md [hunk 2]
```diff
@@ -918,6 +919,39 @@ Also see [push notifications](#push-notifications)
...
+* [Carnival](http://www.carnival.io)
+* [Catapush](http://www.catapush.com/)
+* [Netmera](http://www.netmera.com)
+* [OneSignal](https://onesignal.com) - Free.
+* [PushBots](https://pushbots.com/)
+* [Pushwoosh](https://www.pushwoosh.com)
+* [Pushkin](https://github.com/Nordeus/pushkin) - Free and open-source.
+* [Pusher](https://pusher.com/push-notifications) - Free and unlimited.
+
+
```
Comment (lfarah) [inline README.md]: Please remove this extra space

**Comment by d7laungani** [comment]
Fixed issues

File: README.md [hunk 3]
```diff
@@ -919,10 +920,40 @@ Also see [push notifications](#push-notifications)
 * [Nikka](https://github.com/JustaLab/Nikka) - A super simple Networking wrapper that supports many JSON libraries, Futures and Rx :large_orange_diamond: ⌚
 
 #### Email
-
```
Comment (lfarah) [inline README.md]: Can you remove this change?

Comment (d7laungani) [inline README.md]: I added a space between the email tag and its items. Sorry for making this a complicated pull request.

**Comment by lfarah** [comment]
This is kinda of a big change. @dkhamsing @vsouza @victorgama can one of you also approve?

**Comment by dkhamsing** [comment]
This changes removed the link to providers.. is that ok

**Comment by victorgama** [comment]
Indeed! @d7laungani Can you please include a link to the `Push Notification Providers` subsection to the content index?

**Comment by d7laungani** [comment]
@victorgama  done

**Pull request merged**